app/assets/stylesheets/styles/components/crops/components/_crop_info_container.scss

```diff
@@ -31,6 +41,18 @@
   }
 }
 
+.small-crop img {
```

Comment:
consider using the class that's already there instead of a element-level declaration.